proto/section_response.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
+
+import "content.proto";
+
+option java_package = "de.stroeer.model.wrapper.generated";
+option java_multiple_files = true;
+
+message SectionResponse {
+  repeated Stage stages = 1;
+}
+
+message Stage {
+  ArticleGroup main = 1;
+  ArticleGroup sib = 2;
+}
+
+message ArticleGroup {
```

Comment:
I don't like this name, but I don't have a better one. What happened to the idea to introduce a `Teaser` model on `Section`'s?